contracts/src/protocol/OmniAVS.sol

```diff
@@ -1,106 +1,169 @@
 // SPDX-License-Identifier: GPL-3.0-only
-pragma solidity 0.8.12;
+pragma solidity =0.8.12;
```

Comment:
can we change this to `0.8.23` now? We're not importing contracts with this requirement anymore